lib/vanilla-masker.js

```diff
@@ -149,6 +151,10 @@
     }
     masked = masked.replace(clearDelimiter, "");
     masked = masked.length ? masked : "0";
+    var signal = "";
+    if(opts.showSignal === true) {
+      signal = value < 0 || value.startsWith('-') ? "-" : (opts.showPositiveSignal === true ? "+" : "");
```

Comment:
Or we could never show the `+` signal, as you said. It would be simpler indeed, and I'm not sure someone will ever miss the feature 😉